coalib/processes/BearRunning.py

```diff
@@ -5,8 +5,8 @@
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
+from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
```

Comment:
don't we want to remove LogMessage as well? Does builtin logging care if we do it across processes? If it doesn't we can just use builtin logging and remove the whole log printer thread right?